src/app/components/shared/demo-page.component.html

```diff
@@ -26,7 +26,13 @@ <h4 *ngIf="skyuxModulesForDisplay">
     <stache-page-anchor>
       Installation
     </stache-page-anchor>
+    <p>
+      To use <span *ngFor="let moduleName of skyuxModulesForDisplay"><stache-code>{{ moduleName }}</stache-code></span>, you must first install the module's NPM package to set it up as a dependency. In the command prompt, run the following command from your SPA's root directory:
```

Comment:
I'm wondering if we could clarify that you use exports from the module, not the module itself.  Things like pipes, components, and services. 

Perhaps `To use any exports from the {{ skyuxModulesForDisplay }} module, you must install it as a dependency to your project.  In the command prompt, run the following command in your SPA's root directory:`

Reply:
Makes sense.